server/server.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+# Example api route
+@app.route("/members")
+def members():
+  return {"members": ["Member1", "Member2", "Member3"]}
+
+if __name__ == "__main__":
+  app.run(debug=True) # since this is in development mode
```

Comment:
Can we use environment variables to create a boolean for the debug flag? either `DEBUG=1` or consider another environment variable like the NODE_ENV var that establishes `production` vs `development`?